### PR TITLE
Resolve a number of outstanding FAQ issues

### PR DESCRIPTION
**Pull request by alilleybrinker**
This resolves a number of the items in #886. In particular:

__“Is this project controlled by Mozilla?“__

- [x] Update the number of unique contributors.
- [x] Should the yearly review cycle be in here?

__“In which projects is Mozilla using Rust?“__

- [x] Update to include info about Rust use in Firefox.

__“What examples are there of large Rust projects?“__

- [x] See if there are larger projects which could warrant adding.

__“How can I try Rust easily?”__

- [x] “playpen” -> “playground”

__“How do I get help with Rust issues?”__

- [x] Remove /r/rust, because it’s not an official channel.

__“Why has Rust changed so much over time?”__

- [x] Link to Rust’s specific stability policy.

__“How fast is Rust?“__

- [x] Include links to more benchmarks and comparisons?

__“Why curly braces? Why can’t Rust’s syntax be like Haskell’s or Python’s?”__

- [x] Remove reference to the complexity of the parser in the compiler.

__“What is ‘monomorphisation’?“__

- [x] We’re using the UK English spelling here. Do we want to be? Do we have a policy for this?

__“What the deal with `unwrap()` everywhere?“__

- [x] The existence of the `?` should probably be mentioned.

__“How do I get an error when I try to run example code that uses the `try!` macro?”__

- [x] The use of `try!` (and of course `?` by extension) inside of `main` has been accepted as an RFC. This should be mentioned.

### CODE REVIEW

**Comment by aturon** [comment]
Hi @AndrewBrinker! Sorry this never got followed up on -- we don't have a great review process for the website. Want to rebase, then ping me for review?

**Comment by AndrewBrinker** [comment]
Alright, it should be ready to go now. @aturon

File: en-US/faq.md [hunk 1]
```diff
@@ -64,10 +64,11 @@ Rust exists as an alternative that provides both efficient code and a comfortabl
...
+As far as [project governance](https://github.com/rust-lang/rfcs/blob/master/text/1068-rust-governance.md) goes, Rust is managed by a core team that sets the vision and priorities for the project, guiding it from a global perspective. There are also subteams to guide and foster development of particular areas of interest, including the core language, the compiler, Rust libraries, Rust tools, and moderation of the official Rust communities. Designs in each of these areas are advanced through an [RFC process](https://github.com/rust-lang/rfcs). For changes which do not require an RFC, decisions are made through pull requests on the [`rustc` repository](https://github.com/rust-lang/rust).
+
+Long term planning is guided by a [yearly roadmap](https://github.com/brson/rfcs/blob/north-star/text/0000-north-star.md), with specific milestones set each year.
```
Comment (Mark-Simulacrum) [inline en-US/faq.md]: This should link to the merged RFC instead, which I believe is https://github.com/rust-lang/rfcs/blob/master/text/1728-north-star.md.

File: en-US/faq.md [hunk 2]
```diff
@@ -118,13 +119,12 @@ There are several ways. You can:
...
+Rust started with a goal of creating a safe but usable systems programming language. In pursuit of this goal it explored a lot of ideas, some of which it kept (lifetimes, traits) while others were discarded (the typestate system, green threading). Also, in the run up to 1.0 a lot of the standard library was rewritten as early designs were updated to best use Rust's features and provide quality, consistent cross-platform APIs. Now that Rust has reached 1.0, the language is [guaranteed to be "stable"](https://blog.rust-lang.org/2014/10/30/Stability.html); and while it may continue to evolve, code which works on current Rust should continue to work on future releases.
```
Comment (Mark-Simulacrum) [inline en-US/faq.md]: Hm, I'm not sure about putting stable in quotes -- I think linking to the stability policy is good (but I think there's an RFC which covers this better? I wasn't able to quickly find it, but maybe it's https://github.com/rust-lang/rfcs/blob/master/text/1105-api-evolution.md? @aturon can probably help more here...)

**Review by Mark-Simulacrum — COMMENTED** [review]
I think this looks good overall, but left some initial comments.

**Comment by steveklabnik** [comment]
@AndrewBrinker ping! Got anything to say about @Mark-Simulacrum 's comments?

**Pull request closed without merging**